### PR TITLE
Fix problems found by codespell

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: _posts/2019-03-16-podman-install.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ tags: containers, images, docker, buildah, podman, oci
 # Installation of Podman to Run Docker Container - Part 1
 ## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
 
-A new article about how Opvizor installed [Podman to run Docker containers](https://www.opvizor.com/installation-of-podman-to-run-docker-container-part-1?sp_url=6k5w).  This blog entry at Opvizor looks into their installation process and their early take aways on Podman.
+A new article about how Opvizor installed [Podman to run Docker containers](https://www.opvizor.com/installation-of-podman-to-run-docker-container-part-1?sp_url=6k5w).  This blog entry at Opvizor looks into their installation process and their early take always on Podman.
```
Comment (TomSweeneyRedHat) [inline _posts/2019-03-16-podman-install.md]: This one should be backed out.

File: _posts/2019-12-14-new.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ layout: default
 categories: [new]
 ---
 
-Do you want to konw how to setup RHEL 8 to run containers using Podman? Xuegang Jin has a blog post on the [Red Hat Blog](https://www.redhat.com/en/blog) about this very subject,  [Working with Linux containers on RHEL 8 with Podman, image builder and web console](https://www.redhat.com/en/blog/working-linux-containers-rhel-8-podman-image-builder-and-web-console).  In the post Xuegang how you can use Image Builder to create an OS image, how to run containers with Podman, and how to check the host and containers performance using Web Console. 
+Do you want to know how to setup RHEL 8 to run containers using Podman? Xuegang Jin has a blog post on the [Red Hat Blog](https://www.redhat.com/en/blog) about this very subject,  [Working with Linux containers on RHEL 8 with Podman, image builder and web console](https://www.redhat.com/en/blog/working-linux-containers-rhel-8-podman-image-builder-and-web-console).  In the post Xuegang how you can use Image Builder to create an OS image, how to run containers with Podman, and how to check the host and containers performance using Web Console.
```
Comment (TomSweeneyRedHat) [inline _posts/2019-12-14-new.md]:
```suggestion
Do you want to know how to setup RHEL 8 to run containers using Podman? Xuegang Jin has a blog post on the [Red Hat Blog](https://www.redhat.com/en/blog) about this very subject,  [Working with Linux containers on RHEL 8 with Podman, image builder and web console](https://www.redhat.com/en/blog/working-linux-containers-rhel-8-podman-image-builder-and-web-console).  In the post Xuegang shows you how you can use Image Builder to create an OS image, how to run containers with Podman, and how to check the host and containers performance using Web Console. 
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
Thanks for running all of these @rhatdan, and ZOINKS! I guess my spell checking software isn't catchng as much as I thought as a lot of these were mine.

File: _posts/2019-03-16-podman-install.md (hunk 1 above)
Comment (rhatdan) [reply _posts/2019-03-16-podman-install.md]: Should be `takeaways` according to google.

Comment (TomSweeneyRedHat) [reply _posts/2019-03-16-podman-install.md]: hmm, didn't know "takeaways" was valid.  Thx for the fix up.

**Comment by TomSweeneyRedHat** [comment]
LGTM
and will merge it in, thx @rhatdan

**Pull request merged**